examples/react/w3console/src/share.tsx

```diff
@@ -80,9 +112,43 @@ export function SpaceShare (): JSX.Element {
             value={value}
             onChange={onChange}
           />
-          <a className='w3ui-button text-center block w-52 opacity-30' style={{ opacity: downloadUrl !== '' ? '1' : '0.2' }} href={downloadUrl ?? ''} download={downloadName(downloadUrl !== '', value)}>Download UCAN</a>
+          <a className='w3ui-button text-center block w-52' style={{ opacity: downloadUrl !== '' ? '1' : '0.2' }} href={downloadUrl ?? ''} download={downloadName(downloadUrl !== '', value)}>Download UCAN</a>
         </form>
       </div>
+      <div className='mt-16 py-16 border-t border-gray-700'>
+        <Header>Import a space</Header>
+        <p className='mb-2'>Copy and paste your DID to your friend</p>
+        <div className='bg-opacity-10 bg-white font-mono text-sm py-2 px-3 rounded break-words max-w-4xl'>
+          {agent?.did()}
+        </div>
+        <div className='mt-8'>
+          <label className='w3ui-button text-center block w-52'>
+            Import UCAN
+            <input type='file' accept='.ucan,.car,application/vnd.ipfs.car' className='hidden' onChange={(e: ChangeEvent<HTMLInputElement>) => { void onImport(e) }} />
+          </label>
+        </div>
+        {proof !== undefined && (
+          <div className='mt-4 pt-4'>
+            <Header>Added</Header>
+            <div className='max-w-3xl border border-gray-700 shadow-xl'>
+              {proof.capabilities.map((cap, i) => (
+                <figure className='p-4 flex flex-row items-start gap-2' key={i}>
+                  <DidIcon did={cap.with} />
+                  <figcaption className='grow'>
+                    <a href='#' onClick={() => viewSpace(cap.with)} className='block'>
+                      <span className='block text-xl font-semibold leading-5 mb-1'>{proof.facts.at(i)?.space.name ?? 'Untitled Space'}</span>
```

Comment:
I'd split this long line
```suggestion
                      <span className='block text-xl font-semibold leading-5 mb-1'>
                        {proof.facts.at(i)?.space.name ?? 'Untitled Space'}
                      </span>
```